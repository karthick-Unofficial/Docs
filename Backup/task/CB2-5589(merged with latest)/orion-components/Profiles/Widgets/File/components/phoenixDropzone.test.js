"use strict";

var _interopRequireDefault = require("@babel/runtime-corejs3/helpers/interopRequireDefault");
var _react = _interopRequireDefault(require("react"));
var _PhoenixDropzone = _interopRequireDefault(require("./PhoenixDropzone"));
var _enzyme = require("enzyme");
describe("PhoenixDropzone", function () {
  it("renders", function () {
    var wrapper = (0, _enzyme.shallow)( /*#__PURE__*/_react["default"].createElement(_PhoenixDropzone["default"], null));
    expect(wrapper).toMatchSnapshot();
  });
});