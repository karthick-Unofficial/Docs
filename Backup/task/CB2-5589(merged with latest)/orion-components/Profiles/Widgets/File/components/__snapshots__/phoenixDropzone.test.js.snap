// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoenixDropzone renders 1`] = `
<div>
  <Dropzone
    disabled={false}
    getFilesFromEvent={[Function]}
    maxFiles={0}
    maxSize={Infinity}
    minSize={0}
    multiple={true}
    noClick={false}
    noDrag={false}
    noDragEventsBubbling={false}
    noKeyboard={false}
    onDrop={[Function]}
    preventDropOnDocument={true}
    style={
      Object {
        "display": "none",
      }
    }
    useFsAccessApi={true}
    validator={null}
  >
    <Component />
  </Dropzone>
  <ForwardRef(Button)
    color="primary"
    onClick={[Function]}
    variant="text"
  >
    uploadFiles
  </ForwardRef(Button)>
</div>
`;
